Examples/Tests/reduced_diags/PICMI_inputs_loadbalancecosts.py

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env python3
+
+from pywarpx import picmi
+
+# Number of time steps
+max_steps = 3
+
+# Number of cells
+nx = 128
+ny = 32
+nz = 128
+
+# Physical domain
+xmin = 0.
+xmax = 4.
+ymin = 0.
+ymax = 4.
+zmin = 0.
+zmax = 4.
+
+# Create grid
+grid = picmi.Cartesian3DGrid(
+    number_of_cells = [nx, ny, nz],
+    warpx_max_grid_size=32,
+    lower_bound = [xmin, ymin, zmin],
+    upper_bound = [xmax, ymax, zmax],
+    lower_boundary_conditions = ['periodic', 'periodic', 'periodic'],
+    upper_boundary_conditions = ['periodic', 'periodic', 'periodic'],
+    #lower_boundary_conditions_particles = ['absorbing', 'absorbing', 'absorbing'],
+    #upper_boundary_conditions_particles = ['absorbing', 'absorbing', 'absorbing']
+)
+
+# Electromagnetic solver
+solver = picmi.ElectromagneticSolver(
+    grid = grid,
+    method = 'Yee',
+    cfl = 0.99999
+)
+
+# Particles
+electrons = picmi.Species(
+    particle_type = 'electron',
+    name = 'electrons',
+    initial_distribution = picmi.UniformDistribution(
+        density=1e14, rms_velocity=[0]*3,
+        upper_bound=[xmax, ymax, 1.0]
+    )
+)
+layout = picmi.GriddedLayout(
+    n_macroparticle_per_cell=[1, 1, 1], grid=grid
+)
+
+# Reduced diagnostic
+reduced_diag = picmi.ReducedDiagnostic(
```

Comment:
This is a picky comment you can ignore if you wish, but there is inconsistent formatting in the use of spaces around equals signs, and with one argument per line.